Update white-paper.en.md Correction of amounts for fees (#96)
DeFiCh/defichain.com: content/en/white-paper.en.md

#### Masternodes for Staking

> To run a masternode (staking node), stakers must hold a fixed amount of DFI, 20,000. Masternodes on DeFiChain participate in active transaction validations and block creations. The staking amount is intended to be lowered with the stability and maturity of blockchain to encourage further decentralization.

Reviewer: ```suggestion
To run a masternode (staking node), stakers must hold a fixed amount of 20,000 DFI. Masternodes on DeFiChain participate in active transaction validations and block creations. The staking amount is intended to be lowered with the stability and maturity of blockchain to encourage further decentralization.
```

To be able to apply a penalty to stakers who double-sign, DeFiChain has to disallow immediate withdrawing of stake. Thus, when a deactivation transaction is confirmed, DeFiChain requires 3000 blocks to pass. At a block time of 30 seconds, 3000 blocks is equivalent to 25 hours.

> The double-sign penalty is 10 times the block rewards, deducted from the collateral. This also disqualifies the stakers from further staking immediately. The staker wanting to get back to staking has to re-put in fresh stake UTXO of 20,000 DFI. Running the official DeFiChain node does not cause any unintentional or accidental double-sign. Double-sign happens only in cases of malicious intent.

Reviewer: ```suggestion
The double-sign penalty is 10 times the block rewards, deducted from the collateral. This also disqualifies the stakers from further staking immediately. The staker who wants to get back to staking has to deposit a fresh stake UTXO of 20,000 DFI. Running the official DeFiChain node does not cause any unintentional or accidental double-sign. Double-sign happens only in cases of malicious intent.
```

### Masternodes

> DeFi is a Proof of Stake blockchain. Initially, 1,000,000 DFI, now, Status Q1 2022, 20,000 DFI allow the owner to own a staking node. The returns for staking will decrease over time, as the volume and number of transactions compensates for the reduction in per-transaction staking rewards.

Reviewer: ```suggestion
DeFi is a Proof of Stake blockchain. Initially, 1,000,000 DFI allows the owner to own a staking node. Today, the amount has been reduced to 20,000 DFI for ownership of a staking node. The returns for staking will decrease over time, as the volume and number of transactions compensate for the reduction in per-transaction staking rewards.
```
